<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>L2LDistanceWidget</class>
 <widget class="QWidget" name="L2LDistanceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>680</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBoxLine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>311</width>
     <height>151</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>边1</string>
   </property>
   <widget class="QComboBox" name="comboBoxLine1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>直线任务:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>直线结果:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxLine1Result">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>520</y>
     <width>311</width>
     <height>151</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>边2</string>
   </property>
   <widget class="QComboBox" name="comboBoxLine2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>直线任务:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>直线结果:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxLine2Result">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>100</y>
     <width>341</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>判断条件数据类型</string>
   </property>
   <widget class="QRadioButton" name="radioButton_phy">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>物理值</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_pix">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>像素值</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_XDataRange">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>140</y>
      <width>311</width>
      <height>161</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="title">
     <string>距离</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="MaxValidData">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>---</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lineEdit_Dist">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="MinValidData">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>---</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>下限</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>81</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>计算值</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>上限</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxCalculation">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>520</y>
     <width>341</width>
     <height>151</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>计算结果</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="btnCalculation">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>计算</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_AutoSet">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>自动设置</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
